src/__experimental__/components/search/search.component.js

```diff
@@ -14,6 +14,7 @@ const Search = ({
   defaultValue,
   onChange,
   onClick,
+  onMouseDown,
```

Comment:
this prop is never used, I'm not sure we need it tbh as we only want the mouseDown event handler to stop the focus behaviour

Reply:
It's gone on the next commit 👍🏻